lib/updater.js

```diff
@@ -106,10 +106,10 @@ function _handleCfChunk(cf_chunk, cf_arg, cf_cb) {
          * with the given VM.
          */
         cf_arg.server = { server_uuid: vm.server_uuid };
-        if (vm.state === 'running') {
+        if (vm.state === 'running' || vm.state === 'stopped') {
             _mapAndCacheVm(vm, 'Changefeed', cache, log, cf_cb);
             return;
-        } else if (vm.state === 'stopped' || vm.state === 'destroyed') {
+        } else if (vm.state === 'destroyed') {
             if (cache.vms.has(vm.uuid)) {
                 cache.vms.delete(vm.uuid);
                 log.trace(
```

Comment:
This behavior needs to be conditional on the config. The current behavior always acts as if the config option is enabled when a zone transitions from running to stopped.

Reply:
Will change